Tests/DistTestCore/Logs/StatusLog.cs

```diff
@@ -26,7 +28,7 @@ public void ConcludeTest(string resultStatus, TimeSpan testDuration, Dictionary<
         public void ConcludeTest(string resultStatus, string testDuration, Dictionary<string, string> data)
         {
             data.Add("timestamp", DateTime.UtcNow.ToString("o"));
-            data.Add("runid", NameUtils.GetRunId());
+            data.Add("runid", deployId);
```

Comment:
key is still "runid". Should it be "deployid"?

Reply:
Yeah, I was delaying changing the actual label to sync up with @veaceslavdoina, maybe now is the time.